src/js/player.js

```diff
@@ -4133,6 +4133,7 @@ class Player extends Component {
     }
 
     this.language_ = String(code).toLowerCase();
+    this.trigger('languagechange');
```

Comment:
Also, we want to add an event jsdoc here, see https://github.com/videojs/video.js/blob/cb70ea84a1b3bb02b81af16628fbd617700ca144/src/js/player.js#L572-L578 for an example
and then add a `@fires Player#langaugechange` in the jsdoc comment for the method.